spec/requests/vms_spec.rb

```diff
@@ -2139,6 +2139,68 @@ def query_match_regexp(*tables)
     end
   end
 
+  describe "POST /api/vms/:id" do
+    let(:ems) { FactoryBot.create(:ems_openstack) }
+    let(:tenant) { FactoryBot.create(:cloud_tenant_openstack, :ext_management_system => ems) }
+    let(:floating_ip) { FactoryBot.create(:floating_ip_openstack, :ext_management_system => ems.network_manager, :cloud_tenant => tenant) }
+
+    it 'associates a floating ip to the vm' do
+      vm = FactoryBot.create(:vm_openstack, :ems_id => ems.id, :cloud_tenant => tenant)
+      api_basic_authorize action_identifier(:vms, :associate)
+
+      post(api_vms_url, :params => {:action => 'associate', :resource => {:id => vm.id, :floating_ip => floating_ip}})
+
+      expect(response).to have_http_status(:ok)
+      task_id = response.parsed_body["task_id"]
+      expected = {
+        "results" => [{
+          "message"   => "Associating resource to Vm id: #{vm.id} name: '#{vm.name}",
+          "success"   => true,
+          "task_id"   => "#{task_id}",
```

Comment:
```suggestion
          "message" => "Associating resource to Vm id: #{vm.id} name: '#{vm.name}",
          "success" => true,
          "task_id" => "#{task_id}",
```

It is warning about the hash alignment because you have extra padding that isn't needed